macosx/TorrentCell.mm

```diff
@@ -126,6 +129,16 @@ - (NSRect)iconRectForBounds:(NSRect)bounds
     return NSMakeRect(NSMinX(bounds) + PADDING_HORIZONTAL, ceil(NSMidY(bounds) - imageSize * 0.5), imageSize, imageSize);
 }
 
+- (NSRect)actionRectForBounds:(NSRect)bounds
+{
+    BOOL const minimal = [self.fDefaults boolForKey:@"SmallView"];
+
+    NSRect iconRect = [self iconRectForBounds:bounds];
+    iconRect.origin.x += minimal ? 0.0 : GROUP_INSET_REG * 3;
```

Comment:
Why multiplying by 3?

Reply:
padding on either side of the pill + the pill width

Reply:
Then it should be 2 * padding + pill width, no?